src/com/palmergames/bukkit/towny/db/TownyDatabaseHandler.java

```diff
@@ -1053,12 +1010,8 @@ public void renamePlayer(Resident resident, String newName) throws AlreadyRegist
 			List<Resident> toSaveResident = new ArrayList<>(getResidents());
```

Comment:
Although this isn't part of the diff, I still have the same comment as the others, why a new arraylist if that arraylist already exists?